src/config_file.c

```diff
@@ -431,6 +431,25 @@ gboolean load_config(const gchar *filename, RaucConfig **config, GError **error)
 				goto free;
 			}
 			g_key_file_remove_key(key_file, groups[i], "resize", NULL);
+
+			if(g_strcmp0(slot->type, "boot-mbr-switch") == 0) {
```

Comment:
Not yet checked by uncrustify but de-facto style in the code: please add a space after `if`